tests/components/fastdotcom/test_services.py

```diff
@@ -0,0 +1,49 @@
+"""Test the Fast.com integration services."""
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components.fastdotcom.const import DEFAULT_NAME, DOMAIN, SERVICE_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+
+from tests.common import MockConfigEntry
+
+
+async def test_services(hass: HomeAssistant) -> None:
+    """Test the service."""
+    config_entry = MockConfigEntry(
+        domain=DOMAIN, unique_id="UNIQUE_TEST_ID", title=DEFAULT_NAME
+    )
+    config_entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.fastdotcom.coordinator.fast_com", return_value=5.0
+    ):
+        await hass.config_entries.async_setup(config_entry.entry_id)
+        await hass.async_block_till_done()
+        await hass.services.async_call(DOMAIN, SERVICE_NAME, blocking=True)
+
+    state = hass.states.get("sensor.fast_com_download")
+    assert state is not None
+    assert state.state == "5.0"
```

Comment:
I'd assert what the state is before the service call too. It should change, otherwise the test isn't testing anything.